data/json/items/ammo.json

```
@@ -258,7 +258,7 @@
     "category": "spare_parts",
     "price": 2000,
     "price_postapoc": 250,
-    "name": { "str": "duct tape" },
+    "name": { "str": "duct tape", "str_pl": "duct tape rolls" },
```

```suggestion
    "name": { "str": "duct tape", "str_pl": "rolls of duct tape" },
```
Check to see if you can concatenate `name` like this.

---

Are you referring to adding "rolls" for a plural? In which position?

---

Yeah, switching "duct tape rolls" for "rolls of duct tape."